app/helpers/markdown_helper.rb

```diff
@@ -1,36 +1,42 @@
-module MarkdownHelper
+# frozen_string_literal: true
 
+module MarkdownHelper
   def markdown(content)
-    md = Redcarpet::Markdown.new(FlavoredMarkdown, MARKDOWN_OPTS)
-    Sanitize.fragment(md.render(content), EVIL_TAGS).html_safe
+    renderer = FlavoredMarkdown.new(**MARKDOWN_RENDERER_OPTS)
+    md = Redcarpet::Markdown.new(renderer, **MARKDOWN_OPTS)
+    # As the string has been sanitized we can mark it as HTML safe
+    Sanitize.fragment(md.render(content), EVIL_TAGS).strip.html_safe # rubocop:disable Rails/OutputSafety
   end
 
   def strip_markdown(content)
-    md = Redcarpet::Markdown.new(Redcarpet::Render::StripDown, MARKDOWN_OPTS)
+    renderer = Redcarpet::Render::StripDown.new
+    md = Redcarpet::Markdown.new(renderer, **MARKDOWN_OPTS)
     CGI.unescape_html(Sanitize.fragment(CGI.escape_html(md.render(content)), EVIL_TAGS)).strip
   end
 
   def twitter_markdown(content)
-    md = Redcarpet::Markdown.new(TwitteredMarkdown, MARKDOWN_OPTS)
+    renderer = TwitteredMarkdown.new
+    md = Redcarpet::Markdown.new(renderer, **MARKDOWN_OPTS)
     CGI.unescape_html(Sanitize.fragment(CGI.escape_html(md.render(content)), EVIL_TAGS)).strip
   end
 
   def question_markdown(content)
-    md = Redcarpet::Markdown.new(QuestionMarkdown.new, MARKDOWN_OPTS)
-    Sanitize.fragment(md.render(content), EVIL_TAGS).html_safe
+    renderer = QuestionMarkdown.new
+    md = Redcarpet::Markdown.new(renderer, **MARKDOWN_OPTS)
+    # As the string has been sanitized we can mark it as HTML safe
+    Sanitize.fragment(md.render(content), EVIL_TAGS).strip.html_safe # rubocop:disable Rails/OutputSafety
   end
 
   def raw_markdown(content)
-    md = Redcarpet::Markdown.new(Redcarpet::Render::HTML, RAW_MARKDOWN_OPTS)
+    renderer = Redcarpet::Render::HTML.new(MARKDOWN_RENDERER_OPTS)
+    md = Redcarpet::Markdown.new(renderer, RAW_MARKDOWN_OPTS)
```

Comment:
1. i think we can get rid of `RAW_MARKDOWN_OPTS`, as it's mostly the same as `MARKDOWN_OPTS` now
2. why is this not `**`d?